### DISCUSSION TITLE
Reuse a function across pipelines

### DISCUSSION THREAD

**Discussion by lucasjamar** [kedro-org/kedro#1257] [Idea]
Hello,

I have a function made of a post request that I use in both my first pipeline ( called data_collection) and last pipeline (called send_results). What would be the best practice to make this function to both pipelines?

Thanks!

Lucas Jamar

**Comment by datajoely** [comment]
Hi @lucasjamar I think this is a good use case for a 1 node modular pipeline. Part of the concept of modular pipelines encourages users to think of pipelines more like complex nodes that can be paramterised.

We have some [docs on the subject here](https://kedro.readthedocs.io/en/latest/06_nodes_and_pipelines/03_modular_pipelines.html)

**Reply by lucasjamar** [reply]
Ill try that out. thanks for the tip :)
